test_files/structs.d

```diff
@@ -26,28 +26,26 @@ struct A
 
 struct C
 {
-    struct
+    struct _Anonymous_0
     {
         int x;
         int y;
     }
```

Comment:
This doesn't work. Now it's not possible to access `x` or `y`. Ideally it should be translated to:

```d
struct C
{
    struct _Anonymous_0
    {
        int x;
        int y;
    }
    
    _Anonymous_0 point;
}
```